src/status/accounts.nim

```diff
@@ -48,3 +48,6 @@ proc reset*(self: AccountModel) =
 
 proc generateAlias*(publicKey: string): string =
   result = status_accounts.generateAlias(publicKey)
+
+proc generateIdenticon*(publicKey: string): string =
+  result = status_accuonts.generateIdenticon(publicKey)
```

Comment:
I'm not sure how it compiled with this typo lol
```suggestion
  result = status_accounts.generateIdenticon(publicKey)
```

Reply:
Lol, that was me undo'ing a change after I've realized I still need the API 

Good catcH! :D